core/modules/installer/installer.manager.inc

```diff
@@ -59,6 +59,7 @@
  * @ingroup forms
  */
 function installer_manager_update_form($form, $form_state, $context) {
+  $installer_settings_config = config('installer.settings');
```

Comment:
Actually, in this loooong function :smile:  it's not only the same config, but also the same config setting: `config_get('installer.settings', 'core_update')`

It this case it would be possible to set a variable and use that twice, for instance:
```
$core_update_enabled = config_get('installer.settings', 'core_update');
```
It's up to you, whatever you think is best.

Reply:
Followed your recommendation and changed the file.